manifest_staging/charts/gatekeeper/Chart.yaml

```diff
@@ -3,8 +3,8 @@ description: A Helm chart for Gatekeeper
 name: gatekeeper
 keywords:
   - open policy agent
-version: v3.1.0
+version: 3.1.1
```

Comment:
Looks like we've dropped the "v" prefix?

Dunno if this is an issue.

Reply:
yes, this was intentional https://github.com/open-policy-agent/gatekeeper/pull/775